guides/source/getting_started.md

```diff
@@ -1510,7 +1510,7 @@ comments on articles.
 
 We're going to see the same generator that we used before when creating
 the `Article` model. This time we'll create a `Comment` model to hold
-reference of article comments. Run this command in your terminal:
+reference to an article. Run this command in your terminal:
```

Comment:
In this guide about associations, I think `reference` has special meaning. `reference` is equivalent to `belongs_to`. So I feel

> `Comment` model to hold reference of article comments

means `Comment` model belongs_to article comments.

If this sentence means that creating `Comment` model to persist article comments, maybe it is better to write that ... create a `Comment` model to hold article comments.